articles/active-directory/saas-apps/sectigo-certificate-manager-tutorial.md

```diff
@@ -14,7 +14,7 @@ ms.author: jeedes
 ---
 # Tutorial: Azure Active Directory integration with Sectigo Certificate Manager
 
-In this tutorial, you learn how to integrate Sectigo Certificate Manager with Azure Active Directory (Azure AD).
+In this tutorial, you learn how to integrate Sectigo Certificate Manager (SCM) with Azure Active Directory (Azure AD).
```

Comment:
@murraymccullighsectigo Hi. Microsoft style is to introduce an acronym only if you will then use the acronym to refer to the term throughout the article. Do you want us to change all subsequent product references from _Sectigo Certificate Manager_ to _SCM_?

Reply:
I considered making that change but decide that would create maybe more churn in the doc than was necessary.  

Sectigo does refer to the product as SCM throughout our own documentation so customers are familiar with that acronym.